modules/redis/redis_test.go

```diff
@@ -0,0 +1,60 @@
+package redis
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/netdata/go-orchestrator/module"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Please add test for `Init` and `Check`. I see only tests for `happy path`, would be nice to have some fail scenarios

Reply:
Yeah I'm definitely not done with the tests here but I'd like to spend more time going through the framework you guys wrote before trying to write a test for `Init` or `Check`. I don't want to end up just testing implementation details like [here](https://github.com/netdata/go.d.plugin/blob/master/modules/rabbitmq/rabbitmq_test.go#L20) or [here](https://github.com/netdata/go.d.plugin/blob/master/modules/rabbitmq/rabbitmq_test.go#L24) for example. I'll try to see if I can come up with some tests for those methods that bring more value. 

Also since we're talking about tests it's starting to be a bit of a burden having to test this against a real Redis/Pika combination. An integration test could bring lots of value here I think. I noticed you guys have a [docker-compose](https://github.com/netdata/go.d.plugin/blob/master/docker-compose.yml), could we potentially use that one to bring a few more containers up? Thoughts?

Reply:
Yes, those tests you mentioned are not good. 

> An integration test could bring lots of value

Impossible to argue 😄